.github/workflows/pr_push.yml

```diff
@@ -1,93 +1,96 @@
+# Checks required for a PR to merge. This workflow mostly call other workflows.
 name: PR/push
 
 on: [push, pull_request]
 
 concurrency:
-    group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
-    cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
 
 permissions:
   contents: read
 
 jobs:
-    FastBuild:
-        name: Fast build
-        strategy:
-            matrix:
-                os: [ubuntu-latest]
-                nproc: ['$(nproc)']
-                pool_scalable: ['ON']
-                disjoint: ['ON', 'OFF']
-                jemalloc: ['ON']
-                include:
-                    - os: windows-latest
-                      nproc: $Env:NUMBER_OF_PROCESSORS
-                      pool_scalable: 'OFF'
-                      disjoint: 'OFF'
-                      jemalloc: 'OFF'
-        runs-on: ${{matrix.os}}
+  FastBuild:
+    name: Fast build
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        nproc: ['$(nproc)']
```

Comment:
I see that sometimes we get nproc in matrix definition and sometimes during cmake call. IMO we shouldn't extend matrix with args that would be defined the same way for all generated jobs. I'm not sure but maybe this will fix the CodeQL warnings about missing configurations like ".github/workflows/codeql.yml:analyze/nproc:$(nproc)/os:ubuntu-latest"

Reply:
But this is related only to building, not testing. OK - let's discuss this offline